src/renderer/components/ReactFlowBox.tsx

```diff
@@ -512,6 +513,15 @@ export const ReactFlowBox = memo(({ wrapperRef, nodeTypes, edgeTypes }: ReactFlo
                     size={1}
                     variant={BackgroundVariant.Dots}
                 />
+                <MiniMap
+                    pannable
+                    zoomable
+                    ariaLabel=""
+                    maskColor="rgb(11, 13, 18, 0.175)"
+                    maskStrokeColor="rgb(255, 255, 255, 1.0"
```

Comment:
This isn't a valid color.

Reply:
Could you elaborate more on how this rgb triplet definition + opacity isn't a valid color?

Reply:
Part of the Minimap Ux, in my experience in other apps,  is the fact that you can quickly navigate around the entire canvas quickly with big strides, so I'm ok with having the scrollZoom factor increment be more than when you are directly scrolling on the canvas but I also see where you're coming from and that this is currently perhaps a little too much so I'll take it back by 50%.

Reply:
> Could you elaborate more on how this rgb triplet definition + opacity isn't a valid color?

It's missing a closing parenthesis

Reply:
Fixed!